projects/plugins/jetpack/extensions/blocks/simple-payments/simple-payments.php

```diff
@@ -51,6 +51,11 @@ function render_block( $attr, $content ) {
 	}
 
 	$simple_payments = Jetpack_Simple_Payments::getInstance();
+
+	if ( ! $simple_payments->is_valid( $attr ) ) {
+		return '';
```

Comment:
Shouldn't we return content here as well, like above?

```suggestion
		return $content;
```

Reply:
No, I think that'll defeat what we're trying to achieve here, but perhaps there is something I'm missing.

If we return the content here it'll show a product which can't be bought, with no error message. Basically it would look like this:
![image](https://user-images.githubusercontent.com/93301/149555153-cee256f0-ed36-4cdf-9834-6ec58d315605.png)

The link that says 'Click here to purchase' won't do anything - it depends upon the the rest of the code in this method to make it work.

It's probably ok to do that above - because people can see something in whatever context it is, and hopefully click through to the post to buy the product. Perhaps even that should be improved, but that's a separate issue I feel.

Reply:
That makes sense! 👍